extension/src/help/openWebHelp.ts

```diff
@@ -133,7 +117,10 @@ enum WebHelpCategory {
 	PROPGETTER = 2,
 	PROPSETTER = 3,
 	FUNCTION = 4,
-	ENUM = 5
+	ENUM = 5,
+	DCLTILE = 6,
+	DCLATT = 7,
```

Comment:
this is added by intent?

Reply:
Yes, these put the TypeScript abstraction in sync with the C# versions.